### PR TITLE
OCD-3523: Remove friendly name field from developer and product POC

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: chpl/chpl-service/pom.xml [hunk 1]
```diff
@@ -470,7 +470,6 @@
                         <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductQmsStandard.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchDetails.java</source>
                         <source>src/main/java/gov/healthit/chpl/domain/CertifiedProductTargetedUser.java</source>
-                        <source>src/main/java/gov/healthit/chpl/domain/Contact.java</source>
```
Comment (andlar) [inline chpl/chpl-service/pom.xml:473]: Does the new domain object "PointOfContact" need to be added here?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/Person.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            hashCode += this.title.hashCode();
+        }
+        return hashCode;
+    }
+
+    public String getFullName() {
+        return fullName;
+    }
+
+    public void setFullName(final String fullName) {
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/Person.java]: Don't need `final` here, right?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/PointOfContact.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        super();
+    }
+
+    public PointOfContact(ContactDTO dto) {
+        super(dto);
+        this.contactId = dto.getId();
+    }
+
+    @Builder
+    public PointOfContact(String fullName, String email, String phoneNumber, String title, Long contactId) {
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/PointOfContact.java:41]: Don't need `final` here either, right?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/ContactEntity.java [hunk 4]
```diff
@@ -69,112 +48,22 @@
...
-    public void setFullName(final String fullName) {
-        this.fullName = fullName;
-    }
-
-    public Long getId() {
-        return this.id;
-    }
-
-    public Date getLastModifiedDate() {
-        return Util.getNewDate(lastModifiedDate);
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/ContactEntity.java:125]: Do we still need the `Util.getNewDate` version of the getters/setters?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/exception/ObjectMissingValidationException.java [hunk 5]
```diff
@@ -2,57 +2,26 @@
...
-    public String getObjectId() {
-        return objectId;
-    }
-
-    public void setObjectId(final String objectId) {
-        this.objectId = objectId;
-    }
-
-    public Date getStartDate() {
-        return Util.getNewDate(startDate);
```
Comment (andlar) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/exception/ObjectMissingValidationException.java:44]: Same question re: util

File: chpl/chpl-service/pom.xml (hunk 1 above)
Comment (kekey1) [reply chpl/chpl-service/pom.xml:473]: No, and I don't understand why. I originally did add them but the mvn task that creates the XSD failed with errors. Removing the objects from here allowed the XSD to get created AND it has the new objects. I could not figure out why.

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/ContactEntity.java (hunk 4 above)
Comment (kekey1) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/ContactEntity.java:125]: I think we agreed to leave these as removed. There are 907 references to the Util.getNewDate method so I do not intend to remove others at this point.

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/PointOfContact.java (hunk 3 above)
Comment (kekey1) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/PointOfContact.java:41]: I don't see "final" here... am I missing something?

Comment (andlar) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/PointOfContact.java:41]: Weird. I was sure I saw that there. Sorry!

**Review by andlar — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**